vignettes/articles/03-run-stress-test.Rmd

```diff
@@ -88,6 +88,9 @@ stress_test_arguments
 ### Understanding the input arguments
 The meaning of each of the detail arguments is as follows:
 
+* `baseline_scenario` holding the name of the baseline scenario.
```

Comment:
Maybe we can extend this to explain that we use a business-as-usual scenario as the baseline scenario?

`baseline_scenario` holds the name of the baseline scenario to be used in the stress test. We use a business-as-usual scenario for the baseline.

Reply:
agree

Reply:
done